workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -496,14 +518,19 @@ for (name in names(res.list)){
   genes.to.label <- res.i[1:10, 'symbol']
 
   mdcat('### M-A plot')
-  print(plotMA.label(res.i, genes.to.label=genes.to.label, col='symbol'))
+  print(plotMA.label(res.i,
+                     fdr.thres=alpha.i,
+                     fc.thres=lfc.thresh.i,
+                     genes.to.label=genes.to.label,
+                     col='symbol')
+  )
```

Comment:
...and maybe plotMA.label can figure it out itself, too?

Reply:
I think I like the generality of the function. It just needs a data.frame to make the plot. We can expand it to also support a `DESeqResults` object, so something like a tryCatch with `metadata()` and use the alpha and lfc cutoffs if any. If not, just keep the present form.

Reply:
So something like this?

- the first arg of plot.MA could be a dataframe or a results object
- If dataframe, fdr and fc thresholds must be supplied if they are to be used
- If results object, then use `metadata()` to get the thresholds
- If both results object and threshold(s) are provided...which one wins?

Reply:
yes, exactly. 

I think the last case is the most tricky, and will involve some decisions. I'm thinking explicitly-specified thresholds win? We can output a warning that there is a mismatch

Reply:
explicit-wins sounds good to me, and a warning would be a nice touch.

Reply:
this also reminds me to update the function with my newer version with uses dplyr piping and cleans it up a lot. I'll add an issue with both these things